public-ui/src/app/views/pm/pm.component.html

```diff
@@ -41,20 +41,20 @@ <h5 class="box-title"> {{selectedGroup.groupName}}</h5>
                          selectedConcept.analyses.measurementValueGenderAnalysis" [genderId]="dbc.FEMALE_GENDER_ID"
                      [chartTitle]="gender.analysisStratumName + ' - ' + gender.countValue.toLocaleString()"
                      [chartType]="selectedConcept.chartType"
-                     [analysis]="selectedConcept.analyses.measurementValueGenderAnalysis"></app-chart>
+                     [analysis]="selectedConcept.analyses.measurementValueGenderAnalysis[0]"></app-chart>
```

Comment:
So on this page, we are only showing the first one? That seems reasonable right now. Have you looked at the data in prod to see if all our ppi data comes in mapped to one unit or what ?

Reply:
yeah, the ppi data except pregnany and wheel chair usage seems to have only one unit.